web/src/main/java/com/bazaarvoice/emodb/web/megabus/resource/MegabusResource1.java

```diff
@@ -0,0 +1,66 @@
+package com.bazaarvoice.emodb.web.megabus.resource;
+
+import com.bazaarvoice.emodb.common.json.JsonStreamingArrayParser;
+import com.bazaarvoice.emodb.sor.api.Coordinate;
+import com.bazaarvoice.emodb.web.resources.SuccessResponse;
+import com.bazaarvoice.megabus.MegabusSource;
+import com.google.common.base.Strings;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.io.InputStream;
+import java.util.Iterator;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+@Path ("/megabus/1")
+@Produces (MediaType.APPLICATION_JSON)
+public class MegabusResource1 {
+
+    private final MegabusSource _megabusSource;
+
+    public MegabusResource1(MegabusSource megabusSource) {
+        _megabusSource = checkNotNull(megabusSource, "megabusSource");
+    }
+
+    /**
+     * Touch a document deliberately to send it to Megabus.
+     */
+    @POST
+    @Path ("_touch/{table}/{key}")
+    public SuccessResponse touch(@PathParam ("table") String table,
+                                 @PathParam ("key") String id) {
```

Comment:
Minor: Change "key" to "id" for consistency.

Reply:
Updated.